blog/2021-08-20-Exploring-Template-Haskell.md

```diff
@@ -0,0 +1,572 @@
+---
+title: Exploring Template Haskell
```

Comment:
You should add a github field here so the blog post refers to Daniela's githu account - see how this is done in other blog posts (those where the author has a github account, e.g.  https://github.com/Holmusk/holmusk.github.io/blob/develop/blog/2019-03-29-experience-summary.md)

Daniela's github handle is merrymeerkat

Reply:
Also the `stack run site rebuild` (see README) is failing due to missing author tag.